### PR TITLE
csv format changes

### PR DESCRIPTION
**Pull request by cg-cnu**
* Changed delimiter from `;` to `,`
* Made the changes in the readme file as well
* Added header info as the first item in the csv file

Thanks! 🙂

### CODE REVIEW

File: sensor_database_detailed.csv [hunk 1]
```diff
@@ -1,457 +1,458 @@
...
+AgfaPhoto,sensor 505-D,"1/2.5"" (~ 5.75 x 4.32 mm)",5.75,4.32,2579,1939
+AgfaPhoto,sensor 505-X,"1/2.5"" (~ 5.75 x 4.32 mm)",5.75,4.32,2579,1939
+AgfaPhoto,sensor 530s,"1/2.5"" (~ 5.75 x 4.32 mm)",5.75,4.32,2579,1939
+AgfaPhoto,sensor 830s,"1/2.5"" (~ 5.75 x 4.32 mm)",5.75,4.32,3262,2453
+Apple,iPhone 4,"1/3.2"" (~ 4.54 × 3.42 mm),4.54,3.42,2592,1936 
+Apple,iPhone 4S,"1/3.2"" (~ 4.54 × 3.42 mm),4.54,3.42,3264,2448
+Apple,iPhone 5,"1/3.2"" (~ 4.54 × 3.42 mm),4.54,3.42,3264,2448
+Apple,iPhone 5S,"1/3"" (~ 4.89 × 3.67 mm),4.89,3.67,3264,2448
+Apple,iPhone 6,"1/3"" (~ 4.89 × 3.67 mm),4.89,3.67,3264,2448
+Apple,iPhone 6 Plus,"1/3"" (~ 4.89 × 3.67 mm),4.89,3.67,3264,2448
```
Comment (daz) [inline sensor_database_detailed.csv]: Bad formatting on these iPhone lines. Need an extra `"` after `mm)`

**Comment by cg-cnu** [comment]
Thanks for the inputs @daz 
* Fixed the quotes in the Apple,iPhone and RaspberryPi 
* Realized that vscode replaces does only 10,000 items at a time. Fixed the rest this time.
* Fixed a typo in the cameraMaker header title.

**Comment by cg-cnu** [comment]
👋 Just a friendly ping! 🙂

**Comment by rigakevin** [comment]
You can add CANON EOS 2000D in the database.

The line to add is : Canon;Canon EOS 2000D;22.3

**Pull request closed without merging**